src/control/cmd/dmg/storage.go

```diff
@@ -344,3 +345,43 @@ func (cmd *vmdIdentifyCmd) Execute(_ []string) error {
 	}
 	return cmd.makeRequest(context.Background(), req)
 }
+
+// ledManageCmd is the struct representing the led-manage storage subcommand.
+type ledManageCmd struct {
+	Reset    vmdResetCmd    `command:"reset" description:"Reset the status LED on a VMD NVMe SSD back to a normal OFF state. Useful for debug or mistakenly set LEDs only, not necessary in a typical use case."`
+	GetState vmdGetStateCmd `command:"get-state" description:"Display the current state of the status LED on a VMD device. Useful for remote debugging."`
+}
+
+// vmdResetCmd is the struct representing the led-manage reset storage subcommand.
+type vmdResetCmd struct {
+	smdQueryCmd
+	UUID string `long:"uuid" description:"Device UUID of the VMD SSD LED to reset to OFF" required:"1"`
```

Comment:
I think LED state is always retrieved from SPDK by the transport ID, so the LED states should be consistent when there are multiple namespaces on single SSD.

I'm just wondering if the command of set/get LED state should be changed to use transport ID instead of device UUID?